### PR TITLE
v1.8 Rancher Desktop Documentation Release

### PR DESCRIPTION
**Pull request by sunilarjun**
Updated the dependency list, uploaded macOS/Linux/Windows images to s3 and updated links in latest versions/chinese version.

### CODE REVIEW

**Review by nunix — COMMENTED** [review]
The content is good. 

Please delete the directories `img/preferences` and `img/ui-main` from `/docs`, `/versioned_docs/version-1.8` and `/versioned_docs/version-latest` as it's a redundancy from the S3 images and are no more used.

**Comment by jandubois** [comment]
I haven't had time to look at this, but in the future please don't mix content changes with structural changes. At the very least they should be separate commits.

Structural changes are things like:
* modifying whitespace / indentation
* everything that is just a global search & replace, e.g. updating version strings in links
* moving files around or duplicating files

During a review it is very hard to find any content edits (or new content) mixed in with structural changes of over 200 files...

**Review by jandubois — CHANGES_REQUESTED** [review]
`helm` and `trivy` have been updated again since this PR was created.

File: i18n/zh/docusaurus-plugin-content-docs/version-latest/bundled-utilities-version-info/v1.8.0.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+amazon-ecr-credential-helper: 0.6.0 <br/>
+docker: 23.0.1 <br/>
+docker-buildx: 0.10.4 <br/>
+docker-compose: 2.16.0 <br/>
+docker-credential-helpers: 0.6.4 <br/>
+helm: 3.11.1 <br/>
+kuberlr: 0.4.2 <br/>
+nerdctl: 1.2.1 <br/>
+trivy: 0.38.1 <br/>
```
[the remark this answers is not in the document]
Comment (sunilarjun) [reply i18n/zh/docusaurus-plugin-content-docs/version-latest/bundled-utilities-version-info/v1.8.0.md:9]: I'll go ahead and get these updated thank you Jan, also I checked `trivy` again and they've just released a 0.38.3 version today, I will get this updated asap!

**Comment by sunilarjun** [comment]
The most recent commit addresses PR 162(https://github.com/rancher-sandbox/docs.rancherdesktop.io/pull/162) so as to not create a merge conflict. The application.md page has been updated with new verbiage/images for versions next, latest, and v1.8.

**Review by gunamata — APPROVED** [review]
Tested the PR locally and it looks good to me, except for the double shadowed macOS images for 1.7 version docs, which we need to investigate after the release.

**Review by jandubois — APPROVED** [review]

**Pull request merged**